actors/account/src/types.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
```

Comment:
Delete these comments, or at least the copyright line.

We don't have consistency on the SPDX line. IMO we should either strip them or implement tooling to enforce them, but that can wait.